apps/condo/lang/en/en.json

```diff
@@ -260,6 +261,38 @@
   "callRecord.savedNotification.descriptionLink": "call records log",
   "callRecord.saveErrorNotification.message": "Call not saved",
   "callRecord.saveErrorNotification.description": "An error occurred while saving the call",
+  "documents.title": "Documents",
+  "documents.uploadDocumentsModal.title": "Upload Documents",
+  "documents.uploadDocumentsModal.category.message": "Select the document category for upload - it will help to find them faster in the list",
```

Comment:
```suggestion
  "documents.uploadDocumentsModal.category.message": "Select document category for upload – it will make searching documents easy and fast",
```